homework-submission/week1/src/TodoList.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import todoItems from './todoItems.js';
+
+class TodoList extends Component {
+
+  todoItem = todoItems.map((item, i) => <li key={i}>{ "Task: " + item.description + " deadLine: " + item.deadLine}</li>);
```

Comment:
Here too I think here is a bit more important than the previous one.
so if like I think it would be better for readers more than a long nice line: :slightly_smiling_face: 
```javascript
todoItem = todoItems.map((item, i) => (
    <li key={i}>
      { "Task: " + item.description + " deadLine: " + item.deadLine}
    </li>
  ));
```

Reply:
Indeed it is more readable. Changes are made.